src/support.cpp

```diff
@@ -1013,7 +1013,25 @@ void AreaSupport::generateSupportAreasForMesh(SliceDataStorage& storage,
                 layer_above = &empty;
                 layer_this = layer_this.unionPolygons(storage.support.supportLayers[layer_idx].support_mesh);
             }
-            layer_this = AreaSupport::join(storage, *layer_above, layer_this, smoothing_distance).difference(model_mesh_on_layer);
+            layer_this = AreaSupport::join(storage, *layer_above, layer_this, smoothing_distance);
+            const int layer_size = layer_this.size();
```

Comment:
You should give this a more descriptive name, it isn't clear a few lines down what the difference is between layer\_size and layer\_this.size()